packages/core/src/JWEditor.ts

```diff
@@ -24,6 +25,7 @@ export class JWEditor {
     vDocument: VDocument;
     debugger: OwlUI;
     parser = new Parser();
+    _platform = navigator.platform.match('/Mac/') ? 'mac' : 'pc';
```

Comment:
I'd make it public, especially considering you're setting it directly in the tests.

Reply:
1) I'm setting it in test for mocking purpose.
2) Changing the platform after the editor has been initialized could lead to inconsistencies as it is used only to load shortcuts.
If we need to change the platform, it should be done with the config or a setter.
Changing the platform should reset the shortcuts.